### ISSUE TITLE
No crate-level doccomment

### ISSUE THREAD

**Issue by vi** [Lucretiel/gridly-rs#1]
It's not obvious what gridly does from [the root doc page](https://docs.rs/gridly/0.9.0/gridly/index.html).

**Comment by Lucretiel** [comment]
Yeah, that's one of the many things I need to take care of before 1.0. The short version is that the `grid` module provides traits for describing grids in terms of boundaries and lookups, and the rest of the crate has supporting types like `Location` for indexing into a grid, `Vector` for doing arithmetic between locations, etc.
